docs/jenkins.md

```diff
@@ -0,0 +1,283 @@
+# Jenkins
+
+The Foreman project maintains its own Jenkins instance for continuous integration at http://ci.theforeman.org/
+
+It runs the following types of tests:
+
+* Unit tests on the main develop and recent stable branches of core Foreman projects
+* Pull request tests against core Foreman projects
+* Unit and pull request tests for many Foreman plugins
+* Nightly and release package building and release processes
+* Package smoke and integration testing ("systest")
```

Comment:
I'd drop "systest" as we don't use that word anywhere anymore.

Should list also mention Katello and Pulpcore individually?